libraries/mariadb_helper.rb

```diff
@@ -166,11 +166,17 @@ def scl_packages_names(os_platform, mariadb_version)
     # Helper to determine names of mariadb packages provided by MariaDB
     # @param [String] os_platform Indicate operating system type, e.g. centos
     # @param [String] version Indicate requested version of mariadb
-    def mariadb_packages_names(os_platform, version)
+    def mariadb_packages_names(os_platform, os_version, version)
       if %w(debian ubuntu).include?(os_platform)
-        { 'devel' => 'libmariadbclient-dev',
-          'client' => "mariadb-client-#{version}",
-          'server' => "mariadb-server-#{version}" }
+        if Gem::Version.new(os_version) >= Gem::Version.new('16.00')
```

Comment:
It's quick here to just say node['platform_version'].to_i > 16

Reply:
And to my opinion, it's not a good way to just say that if os_version >16 then we have to change the package name... Because we are in the case debian + ubuntu. They have distinct version number, so it's a non sense to mix it here based on Ubuntu OS version number.

Please do a distinct case for debian and for ubuntu. Thx 😄